src/data.js

```diff
@@ -22,6 +22,7 @@ export const quizData = {
         b: 'var, const, let, function',
         c: 'var, let, const',
       },
+      prize : '$100',
```

Comment:
MoS. We should match the prize with the index. because in our project we are gonna change the order of the questions then.